app/src/app/(payload)/custom.scss

```
@@ -752,6 +718,73 @@ body {
   }
 }
 
+// Taxonomy panels — Manage → Subjects and Manage → Subject grades (PR 3).
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)